medusa/server/web/home/add_shows.py

```diff
@@ -167,6 +167,11 @@ def massAddTable(self, rootDir=None):
                     if not (indexer_id and show_name):
                         (indexer_id, show_name, indexer) = cur_provider.retrieveShowMetadata(cur_path)
 
+                        # unicode encoded special character show names, should be encoded to byte, as the nfo can have a
+                        # show title with unicode encoded special chars. Like naruto shippuuden on tvmaze.
+                        if show_name:
+                            show_name = show_name.encode('utf-8')
+
```

Comment:
@ratoaq2 but `etree.ElementTree(file=xmlFileObj)` reads a xml file. so i don't get why this is the best place.